pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -522,6 +544,46 @@ func (h *NetworkUtilsHandler) DisableTXOffloadChecksum(ifaceName string) error {
 	return nil
 }
 
+func (h *NetworkUtilsHandler) CreateRADaemon(socketPath string, advertisementIfaceName string, ipv6CIDR string, currentRetry int) error {
```

Comment:
In `CreateNDPConnection` you call the iface `bridgeInterfaceName`, here you call it `advertisementIfaceName`. Please be consistent.